cogs/handles.py

```diff
@@ -124,6 +130,12 @@ async def identify(self, ctx, handle: str):
             await discord_.send_message(ctx, data[1])
             ctx.command.reset_cooldown(ctx)
             return
+        # 2 discord users setting same handle
+        handles = list(filter(lambda x: x[2] == handle, self.db.get_all_handles(ctx.guild.id)))
```

Comment:
Move this below [this](https://github.com/pseudocoder10/Lockout-Bot/blob/master/cogs/handles.py#L129) line. This is because handle is case sensitive.

Suppose someone's handle is `abcd`, then `.handle identify Abcd` will not be handled by this check.